src/components/Map/index.jsx

```diff
@@ -1,14 +1,47 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { MapContainer, TileLayer } from "react-leaflet";
-import styles from "./index.module.scss";
 import "leaflet/dist/leaflet.css";
 
+import styles from "./index.module.scss";
+
 const Map = () => {
+  const [coords, setCoords] = useState(null);
+  const [latitudeState, longitudeState] = coords ?? [];
+
+  const setCurrentCoords = (location) => {
+    if (location)
+      setCoords([location.coords.latitude, location.coords.longitude]);
+  };
+
+  const handleLocationBlocked = (error) => {
+    if (error.code === error.PERMISSION_DENIED) {
+      setCoords([-33.6003, -69.426086]);
```

Comment:
consider store these cords in a constant, I think that it makes the code a little bit easy to read
```suggestion
      setCoords(DEFAULT_COORDS);
```